build.gradle.kts

```diff
@@ -125,6 +118,14 @@ val hivemqOpenApi: Configuration by configurations.creating {
     }
```

Comment:
The attributes here are not necessary currently as we are loading them now only from files.
We could make a "repository" out of the specs folder with proper metadata, but that's something for a different ticket.